README.md

```diff
@@ -34,7 +34,7 @@ Flags:
                              URI on which to scrape kamailio. E.g.
                              "unix:/var/run/kamailio/kamailio_ctl" or
                              "tcp://localhost:2049"
-  -m, --kamailio.methods="tm.stats,sl.stats,core.shmmem,core.uptime"
+  -m, --kamailio.methods="tm.stats,sl.stats,core.shmmem,core.uptime,dlg.stats_active"
```

Comment:
This is the "default" metrics, and as "dlg" requires the dialog module, I think we should not enable this metric by default. Plus, you did not enabled it as a default metric in `main.go` (which is good).